priya/Bootstrap/Assignment1/Bootstrap1.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assign1_bootstrap</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.0/dist/css/bootstrap.min.css" integrity="sha384-B0vP5xmATw1+K9KRQjQERJvTumQW0nPEzvF6L/Z6nronJ3oUOFUFpCjEUQouq2+l" crossorigin="anonymous">
+    <style>
+        .div1{
+            width: 880px; 
+            height: 600px
+        }
+        .div2{
+            width: 850px; 
+            height: 200px; 
+            font-size: 50px;    
+        }
+        .div3{
+            border: 1px solid black; 
+            width: 650px; 
+            height: 40px; 
+            position: absolute; 
+            left:400px;
+        }
+        .div4{
+            border: 1px solid black; 
+            width: 650px; 
+            height:40px; 
+            position: absolute; 
+            margin-top: 60px; 
+            left: 400px;
+        }
+        .div5{
+            border: 1px solid black; 
+            width: 650px; 
+            height:40px; 
+            position: absolute; 
+            margin-top: 120px; 
+            left: 400px;
+        }
+        .div6{
+            border: 1px solid black; 
+            width: 650px; 
+            height:40px; 
+            position: absolute; 
+            margin-top: 180px; 
+            left: 400px;
+        }
+        .div7{
+            width: 150px; 
+            height: 100px;
+            text-align: start;
+        }
+    </style>
+</head>
+<body>
+    <div class="container border-rounded  div1">
+        <div class="jumbotron  bg-success div2">MI-10-MEAN</div>
+        <div class="container border div3 ">Announcements</div>
+            <div class="container border div4">Assignment2-Bootsrap</div>
+            <div class="container border div5" >Assignment1-bootstrap</div>
+            <div class="container border div6">Assignment10-css</div>
+        <div class="jumbotron div7">No work due soon!</div>
+        </div>
```

Comment:
<div>tag indentation is incorrect
```suggestion
</div>
```